dist/modules/pager/pager.spec.js

```diff
@@ -0,0 +1,44 @@
+"use strict";
+
+describe('Pager Controller', function () {
+    beforeEach(module('c2fo.ui'));
+
+    var controller,
+        scope;
+
+    beforeEach(inject(function (_$controller_) {
+        // The injector unwraps the underscores (_) from around the
+        // parameter names when matching
+        var $controller = _$controller_;
+
+        scope = {};
+        controller = $controller('PagerController', {$scope: scope});
+    }));
+
+    describe('range()', function () {
+        it('should create a range of iteratees', function () {
+            expect(controller.range(5)).toEqual([0, 1, 2, 3, 4]);
+        });
+
+        it('should create a range of iteratees in reverse', function () {
+            expect(controller.range(5, true)).toEqual([4, 3, 2, 1, 0]);
+        });
+    });
+
+    describe('handlePageChange()', function () {
+        it('should return false on invalid page', function () {
+            expect(controller.handlePageChange(null)).toEqual(false);
+        });
+
+        it('should fire onPageChange() when successful', function () {
```

Comment:
I don't think this test is testing what this suggests.  I'd expect something like this instead: 

```js
onPageChangeSpy = spy(controller, 'onPageChange');

controller.handlePageChange();

expect(onPageChangeSpy).calledOnce; // I can't remember the exact syntax on this one, it's worth a google. 
```

The current test doesn't really test that the `onPageChange` is called. If I changed `handlePageChange` to do nothing, I'm pretty sure this test would pass.